float32[] array # 1d array to be appended as 2d array element